src/gui/macutils/AppKitImpl.mm

```diff
@@ -144,6 +144,13 @@ - (void) removeGlobalMonitor:(id) monitor
 //
 - (bool) enableAccessibility
 {
+    CGImageRef screenshot = CGWindowListCreateImage(
+            CGRectMake(0, 0, 1, 1),
+            kCGWindowListOptionOnScreenOnly,
+            kCGNullWindowID,
+            kCGWindowImageDefault);
+    CFRelease(screenshot);
+
```

Comment:
Update: Actually, please add a comment as to what this does and why it is needed.

Reply:
Done